container/karaf/src/main/filtered-resources/etc/custom.properties

```diff
@@ -795,5 +795,10 @@ org.osgi.framework.system.packages.extra=org.apache.karaf.branding,\
         org.opennms.web.svclayer.model;version=${opennms.osgi.version},\
         org.opennms.web.svclayer.support;version=${opennms.osgi.version},\
         org.opennms.netmgt.icmp;version=${opennms.osgi.version},\
-        org.opennms.netmgt.icmp.proxy;version=${opennms.osgi.version}
+        org.opennms.netmgt.icmp.proxy;version=${opennms.osgi.version},\
+        org.opennms.netmgt.filter.api;version=${opennms.osgi.version},\
```

Comment:
These should be in alphabetical order instead of just appending them to the end.

We could also fix up the the `org.opennms.netmgt.icmp` and `org.opennms.netmgt.icmp.proxy` entries while were at it.